docs/DeveloperGuide.md

```diff
@@ -2,37 +2,615 @@
 
 ## Acknowledgements
 
-{list here sources of all reused/adapted ideas, code, documentation, and third-party libraries -- include links to the original source as well}
+{list here sources of all reused/adapted ideas, code, documentation, and third-party libraries -- include links to the 
+original source as well}
 
```

Comment:
I think it would be great if there was a link to your UG/a brief overview of what this app is programmed to do! It would make it easier see all the following technical details in the bigger picture.